### PR TITLE
task-13

### PR DESCRIPTION
**Pull request by Elfant**

### CODE REVIEW

File: front-end/src/components/Content/Content.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 
 import * as styles from './Content.scss';
 
-const Content: React.FC = () => (
+const Content = () => (
```
Comment (krzysztofbukowski) [inline front-end/src/components/Content/Content.tsx]:
```suggestion
const Content: React.FC = () => (
```

File: front-end/src/components/App.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 
 import Header from './Header/Header';
```
Comment (krzysztofbukowski) [inline front-end/src/components/App.tsx]: Please apply the same pattern like for `Footer` for all components. Add `index.ts`

File: front-end/src/components/Menu/Menu.tsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+
+import style from './Menu.scss';
+
+const Menu: React.FC = () => (
+  <i className={`${style.icon} fas fa-hamburger`}></i>
```
Comment (krzysztofbukowski) [inline front-end/src/components/Menu/Menu.tsx]: Please remove this font-awesome. We're not gonna use this.

File: front-end/src/scss/base.scss [hunk 4]
```diff
@@ -15,7 +15,7 @@
 
   .container {
     width: 100%;
-    max-width: 1000px;
+    max-width: 1200px;
```
Comment (krzysztofbukowski) [inline front-end/src/scss/base.scss]: We can extract this to a variable

File: front-end/index.html [hunk 5]
```diff
@@ -9,4 +10,5 @@
   <body>
     <div id="root"></div>
   </body>
+  <script src="https://kit.fontawesome.com/6356e4e1e9.js" crossorigin="anonymous"></script>
```
[the remark this answers is not in the document]
Comment (Elfant) [reply front-end/index.html]: For icon in menu.

File: front-end/src/components/header/Header.scss [hunk 6]
```diff
@@ -1,24 +1,35 @@
...
+  &Box {
+    align-items: center;
+    display: flex;
+  }
 }
 
-.title {
-  font-size: $font-size;
-  margin: 15px 0; 
+.Search {
```
Comment (krzysztofbukowski) [inline front-end/src/components/header/Header.scss]:
```suggestion
.search {
```
We need to be consistent with naming

**Pull request merged**